Source/SIMPLib/CoreFilters/GenerateColorTable.cpp

```diff
@@ -63,13 +63,13 @@ template <typename T>
 class GenerateColorTableImpl
 {
 public:
-  GenerateColorTableImpl(typename DataArray<T>::Pointer arrayPtr, QVector<float> binPoints,
-                         std::vector<std::vector<double> > controlPoints, int numControlColors, UInt8ArrayType::Pointer colorArray) :
-    m_ArrayPtr(arrayPtr),
-    m_BinPoints(binPoints),
-    m_NumControlColors(numControlColors),
-    m_ControlPoints(controlPoints),
-    m_ColorArray(colorArray)
+  GenerateColorTableImpl(typename DataArray<T>::Pointer arrayPtr, std::vector<float>& binPoints, std::vector<std::vector<double>>& controlPoints, int numControlColors,
```

Comment:
`binPoints` changed from pass by value to pass by reference, and it's not const.

Reply:
Watch `controlPoints` as well.